website/docs/api/top-level.mdx

```diff
@@ -340,6 +340,128 @@ use with the `manual=True` argument in `displacy.render`.
+| `tag`          | Dependency tag of the word. ~~str~~  |
+| `lemma`        | Lemma of the word. ~~Optional[str]~~ |
+
+</Accordion>
+
+<Accordion title="Arc data structure">
+
+| Dictionary Key | Description                                     |
+| -------------- | ----------------------------------------------- |
+| `start`        | Start index. ~~int~~                            |
+| `end`          | End index. ~~int~~                              |
+| `label`        | Label of the arc. ~~str~~                       |
+| `dir`          | Direction of the arc (`left`, `right`). ~~str~~ |
+
+</Accordion>
+
+> #### ENT data structure
+>
+> ```json
+>  {
+>    "text": "But Google is starting from behind.",
+>    "ents": [{"start": 4, "end": 10, "label": "ORG"}],
+>    "title": None
+>  }
+> ```
+
+#### Named Entity Recognition data structure {id="structure-ent"}
+
+| Dictionary Key | Description                                |
+| -------------- | ------------------------------------------ |
+| `text`         | Text of the document. ~~str~~              |
+| `ents`         | List of entities. ~~List[Dict[str, Any]]~~ |
+| `title`        | Title of the visualization. ~~str~~        |
+| `settings`     | Visualization options. ~~Dict[str, Any]~~  |
+
+<Accordion title="Entity data structure">
+
+| Dictionary Key | Description                  |
+| -------------- | ---------------------------- |
+| `start`        | Start index. ~~int~~         |
+| `end`          | End index. ~~int~~           |
+| `label`        | Label of the entity. ~~str~~ |
+| `kb_id`        | Knowledgebase ID. ~~str~~    |
+| `kb_url`       | Knowledgebase URL. ~~str~~   |
+
+</Accordion>
+
+> #### SPAN data structure
+>
+> ```json
+> {
+>   "text": "Welcome to the Bank of China.",
+>   "spans": [
+>     { "start_token": 3, "end_token": 6, "label": "ORG" },
+>     { "start_token": 5, "end_token": 6, "label": "GPE" }
+>   ],
+>   "tokens": ["Welcome", "to", "the", "Bank", "of", "China", "."]
+> }
+> ```
+
+#### Span Classification data structure {id="structure-span"}
+
+| Dictionary Key | Description                               |
+| -------------- | ----------------------------------------- |
+| `text`         | Text of the document. ~~str~~             |
+| `spans`        | List of spans. ~~List[Dict[str, Any]]~~   |
+| `title`        | Title of the visualization. ~~str~~       |
+| `tokens`       | List of tokens. ~~List[str]~~             |
+| `settings`     | Visualization options. ~~Dict[str, Any]~~ |
+
+<Accordion title="Span data structure">
+
+| Dictionary Key | Description                |
+| -------------- | -------------------------- |
+| `start`        | Start index. ~~int~~       |
+| `end`          | End index. ~~int~~         |
+| `start_token`  | Start token. ~~int~~       |
+| `end_token`    | End token. ~~int~~         |
+| `label`        | Label of the span. ~~str~~ |
+| `kb_id`        | Knowledgebase ID. ~~str~~  |
```

Comment:
```suggestion
| `kb_id`        | `KnowledgeBase` ID. ~~str~~  |
```

Should we style this like this and maybe link to the [KnowledgeBase](https://spacy.io/api/kb) docs as well?

Reply:
Ah, yes! I agree

Reply:
Done!